src/2013.ipynb

```diff
@@ -4136,157 +2848,98 @@
-      "\u001b[1;31mAttributeError\u001b[0m: 'NoneType' object has no attribute 'find'"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "projects_final = projects_parser.canonicalize_projects(projects, name_sep='Name:', email_sep='Email:')"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
-   "metadata": {},
+   "execution_count": 14,
+   "metadata": {
+    "scrolled": true
+   },
    "outputs": [
     {
-     "ename": "NameError",
-     "evalue": "name 'projects_final' is not defined",
-     "output_type": "error",
-     "traceback": [
-      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
-      "\u001b[1;31mNameError\u001b[0m                                 Traceback (most recent call last)",
-      "\u001b[1;32m<ipython-input-5-6427e3d633e5>\u001b[0m in \u001b[0;36m<module>\u001b[1;34m\u001b[0m\n\u001b[1;32m----> 1\u001b[1;33m \u001b[0mprojects_final\u001b[0m\u001b[1;33m[\u001b[0m\u001b[1;36m0\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;36m3\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m",
-      "\u001b[1;31mNameError\u001b[0m: name 'projects_final' is not defined"
-     ]
+     "data": {
+      "text/plain": [
+       "106"
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
     }
    ],
    "source": [
-    "projects_final[0:3]"
+    "projects_final[0:3]\n",
+    "len(projects_final)"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "projects_area_combined = projects_parser.apply_transformation(projects_final, \n",
```

Comment:
I've decided to simplify the fields a bit and combine the secondary/primary fields into just one field, like we have in other years.